stock_production_lot_active/views/stock_production_lot.xml

```diff
@@ -0,0 +1,44 @@
+<!-- Copyright 2020 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="stock_production_lot_form_view">
+        <field name="name">stock.lot.form (in stock_production_lot_active)</field>
+        <field name="model">stock.lot</field>
+        <field name="inherit_id" ref="stock.view_production_lot_form" />
+        <field name="arch" type="xml">
+            <field name="display_complete" position="before">
+                <widget
+                    name="web_ribbon"
+                    title="Archived"
+                    bg_color="bg-danger"
+                    invisible="active == True"
```

Comment:
Just a minor comment: on standard Odoo code, this is the way the ribbon is defined:

```suggestion
                    bg_color="text-bg-danger"
                    invisible="active"
```

Not blocking for me, so I approved the PR.